components/Headers/AudioPlayer.js

```diff
@@ -16,6 +16,15 @@ function play() {
     }
   }
 }
+function endFunction() {
+  if (process.browser) {
+    var audio = document.getElementById("a1");
+    var image = document.getElementById("play");
```

Comment:
I think we can use `useRef()` here also.

Reply:
Thanks @akshitarora921 , I did it as intended